sdcm/cluster_aws.py

```diff
@@ -85,7 +85,7 @@ def __init__(self, ec2_ami_id, ec2_subnet_id, ec2_security_group_ids,  # pylint:
                  ec2_user_data='', ec2_block_device_mappings=None,
                  cluster_prefix='cluster',
                  node_prefix='node', n_nodes=10, params=None, node_type=None,
-                 extra_network_interface=False, add_nodes=True):
+                 extra_network_interface=False, add_nodes=True, nodes_smp=[]):
```

Comment:
don't initialize an argument with a mutable list
set it as None, and later in code initialize the list if needed